Assets/Scripts/NavObject.cs

```diff
@@ -0,0 +1,27 @@
+﻿using UnityEngine;
+
+public class NavObject : MonoBehaviour
+{
+    public bool isFollowed = false;
+
+    private int id;
+    public int Id { get => id; set => id = value; }
```

Comment:
Instead of this just use the [System.NonSerialized] tag on a public int id
https://docs.unity3d.com/2017.3/Documentation/ScriptReference/NonSerializable.html